site/src/not-found.md

```diff
@@ -0,0 +1,15 @@
+---
+title: '404: Page not found'
+description: '404: Page not found'
+template: standard
+---
+
+<div class="padding-lg border border-radius-lg">
+	<h1 id="four-zero-four" style="font-size: 12rem" class="font-family-monospace" aria-describedby="four-zero-four-message">404</h1>
+	<p id="four-zero-four-message" class="font-size-h1 color-text-subtle">Shoot! The requested page doesn't exist.</p>
+	<nav class="buttons gap-md margin-top-lg">
+		<a class="button button-primary button-lg button-clear" href="/">Atlas Home</a>
+		<a class="button button-primary button-lg button-clear" href="~/src/atomics/overview.md">Atomics</a>
+		<a class="button button-primary button-lg button-clear" href="~/src/components/overview.md">Components</a>
```

Comment:
Maybe would be nice to add "Patterns" link here as well, but don't feel strongly about it :)

Reply:
There's no overview page for patterns